components/LiveCard.js

```diff
@@ -1,44 +1,102 @@
 import React from 'react';
-import TextStyles from '../constants/TextStyles';
+import { View, StyleSheet } from 'react-native';
 import Svg, { Circle } from 'react-native-svg';
-import { View, Text, StyleSheet } from 'react-native';
+import Animated from 'react-native-reanimated';
 import Colors from '../constants/Colors';
+import Text from '../components/Text';
+
+const {
+  block,
+  Clock,
+  clockRunning,
+  cond,
+  set,
+  spring,
+  startClock,
+  stopClock,
+  Value,
+} = Animated;
+
+function runSpring(clock, value, dest) {
```

Comment:
spring animate the `transform: scale`

Reply:
@dereckquock I wanted a soft blink, like fade in/out blinking (like a recording light) not a bounce 😄

Reply:
i kinda like the bounce 😅 maybe we can work a compromise...lemme see